start.sh

```diff
@@ -83,14 +83,17 @@ function spack-start() {
     if [[ -z $(spack repo list | awk '{print $1" "$2}' | grep "exawind $EXAWIND_MANAGER") ]]; then
       spack -E repo add ${EXAWIND_MANAGER}/repos/exawind
     fi
-    
+
+    export EXAWIND_MANAGER_MACHINE=`spack manager find-machine | awk '{print $2}'`
+    echo "EXAWIND_MANAGER_MACHINE=${EXAWIND_MANAGER_MACHINE}"
+
     if [[ -z $(spack config --scope site blame bootstrap | grep spack-bootstrap-store) ]]; then
       if [[ "${EXAWIND_MANAGER_MACHINE}" == "cee" ]]; then
         spack -E bootstrap add --scope site --trust wind-binaries /projects/wind/spack-bootstrap-store/metadata/binaries
       fi
     fi
 
-    if [[ "${EXAWIND_MANAGER_MACHINE}" == "darwin" && ! -f ${SPACK_CONFIG_USER_PATH}/darwin/compilers.yaml ]]; then
+    if [[ "${EXAWIND_MANAGER_MACHINE}" == "darwin" && ! -f ${SPACK_USER_CONFIG_PATH}/darwin/compilers.yaml ]]; then
```

Comment:
```suggestion
    if [[ "$(spack arch -p)" == "darwin" && ! -f ${SPACK_USER_CONFIG_PATH}/darwin/compilers.yaml ]]; then
```

Reply:
I don't think we need to set this environment variable anymore.

Reply:
I think we should use the find-machine stuff consistently rather than separate methods.

Reply:
We shouldn't need the `cee` one after I get https://github.com/spack/spack/pull/42953 merged. We can actually remove it now if need be.